agreements/EventDefinition/EventJSON/eoI.json

```diff
@@ -0,0 +1,45 @@
+                        "dataType":"date-time",
```

Comment:
Looks like we're missing part of the file here?

Reply:
I think this file is already out of date now too - we probably need to overwrite with @sudhakar-reddy-konda 's latest version

Reply:
I realised a couple of days ago that something was missing. Just not had time to fix. Think I might have time now. Where is the most up to date version? Still in the SQL script but just an updated one?

Reply:
@sudhakar-reddy-konda Updated with what I think is the latest version